BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -2997,7 +2997,7 @@ private void StepRunLoop_Core(bool force = false)
 				}
 
 				bool render = !_throttle.skipNextFrame || (_currAviWriter?.UsesVideo ?? false);
-				Emulator.FrameAdvance(Global.ControllerOutput, render, renderSound);
+				bool new_frame = Emulator.FrameAdvance(Global.ControllerOutput, render, renderSound);

```

Comment:
How hard would it be to compensate for this in the FPS calculation on screen as well?

Reply:
I was thinking about this, and I'm still playing with the idea in my head. With subframe inputs it's not really meant for casual play, so the FPS doens't matter too much, but maybe there is something that can be done to make it have some utility.

Reply:
I just moved the fps calculation to after the frameadvance. Seems to work fine.